debug/testlib.py

```diff
@@ -1210,6 +1210,11 @@ def disable_pmp(self):
             self.gdb.p("$pmpcfg0=0xf")  # TOR, R, W, X
             self.gdb.p("$pmpaddr0=0x%x" %
                     ((self.hart.ram + self.hart.ram_size) >> 2))
+            # If TOR is unsupported (detected via WARL) then use NAPOT instead
+            self.gdb.command("flush regs")
+            if self.gdb.p("$pmpcfg0") != 0xf:
```

Comment:
Can you move this test after writing pmpcfg0, so pmpaddr0 doesn't get written unnecessarily if this mode isn't supported?

Reply:
Sure